### PR TITLE
HANGMAN ! The game

### PR DESCRIPTION
**Pull request by chanson02**
KNOWN ISSUE: Sounds seem to be not working

### CODE REVIEW

File: lib/cli.c [hunk 1]
```diff
@@ -0,0 +1,270 @@
...
+  for (int i=0; i<frame->rows; i++)
+    { free(frame->grid[i]); }
+  free(frame->grid);
+  free(frame);
+  return;
+}
+
+void printAndFreeFrame(Frame* frame) {
+  printFrame(frame);
+  freeFrame(frame);
```
Comment (chanson02) [inline lib/cli.c:47]: Looks like I forgot to return lol

File: lib/cli.c [hunk 2]
```diff
@@ -0,0 +1,270 @@
...
+}
+
+/* return must be free'd with freeFrame */
+Frame* frameFromFile(const char* path) {
```
Comment (chanson02) [inline lib/cli.c:51]: I should probably include where I got this information from, but now I don't remember

File: lib/cli.c [hunk 3]
```diff
@@ -0,0 +1,270 @@
...
+}
+
+
+
+
+
+
+
+/* Function to count characters in string, ignoring escape sequences */
+int getVisibleLength(const char* str) {
```
Comment (chanson02) [inline lib/cli.c:231]: Oops, this doesn't follow my name scheme.

**Review by keegangunkel — APPROVED** [review]
Wow look at this smexy code, you go girl!

**Pull request merged**